contrib/ansible/roles/etcd/tasks/main.yml

```diff
@@ -8,6 +8,11 @@
     - restart etcd
   when: not is_atomic
 
+- name: Clean etcd data directory
+  shell: rm -rf {{ etcd_data_dir }}/*
+  notify:
+    - restart etcd
```

Comment:
ohh boy.. The intention on the ansible stuff here so far was to be as idempotent as possible. Getting your cluster up and running and then any modifications just need another run based on changing variables or nodes.

I would say this is not going to fly.